runtime-tests/src/test/java/com/dylibso/chicory/imports/SpecV1BinaryLeb128HostFuncs.java

```diff
@@ -0,0 +1,24 @@
+package com.dylibso.chicory.imports;
+
+import com.dylibso.chicory.runtime.HostFunction;
+import com.dylibso.chicory.runtime.HostImports;
+import com.dylibso.chicory.runtime.Instance;
+import com.dylibso.chicory.wasm.types.Value;
+import com.dylibso.chicory.wasm.types.ValueType;
+import java.util.List;
+
+public class SpecV1BinaryLeb128HostFuncs {
+    public static HostImports fallback() {
+        return HostImports.builder()
+                .addFunction(
+                        new HostFunction(
+                                (Instance instance, Value... args) -> {
```

Comment:
note to self: `HostFunction` constructor is the only `Host` constructor that doesn't take the two strings for the name as first arguments